tests/app/celery/test_contact_information_tasks.py

```diff
@@ -26,20 +26,13 @@
 notification_id = str(uuid.uuid4())
 
 
-@pytest.fixture(scope='function')
-def notification():
-    recipient_identifier = RecipientIdentifier(
-        notification_id=notification_id, id_type=IdentifierType.VA_PROFILE_ID.value, id_value=EXAMPLE_VA_PROFILE_ID
+def test_should_get_email_address_and_update_notification(client, mocker, sample_template, sample_notification):
```

Comment:
This test doesn't follow our naming conventions. Please update it and the other tests that were modified to have the proper names.

Reply:
We are changing our approach to testing.